src/common/Core/SearchResultItemActionUtility/createShowItemInFileExplorerAction.ts

```diff
@@ -3,7 +3,13 @@ import type { SearchResultItemAction } from "@common/Core";
 /**
  * Creates an action to show the given file in the default file browser.
  */
-export const createShowItemInFileExplorerAction = ({ filePath }: { filePath: string }): SearchResultItemAction => ({
+export const createShowItemInFileExplorerAction = ({
+    filePath,
+    keyboardShortcut,
+}: {
+    filePath: string;
+    keyboardShortcut?: string;
```

Comment:
Why is `keyboardShortcut` optional? Shouldn't we register a shortcut for the Simple File Search as well? It's the only occurrence without a shortcut. The same applies for other actions.

Reply:
I decided to make it optional because if an extension wants to use this action as the default action it doesn't make sense to specify it here as it gets overwritten with "Enter" later on.